src/js/media-captions-menu-button.js

```diff
@@ -123,8 +132,45 @@ class MediaCaptionsMenuButton extends window.HTMLElement {
         this.disable();
       }
     }
-  }
 
+    if (
+      this.hasAttribute('default-showing') && // we want to show captions by default
+    this.#enabledSlot.hidden // and we aren't currently showing them
+    ) {
+      // Make sure we're only checking against the relevant attributes based on whether or not we are using subtitles fallback
+      const subtitlesIncluded = !this.hasAttribute('no-subtitles-fallback');
```

Comment:
```suggestion
      const subtitlesIncluded = !this.hasAttribute('nosubtitlesfallback');
```

Reply:
Since this was a port from media-captions-button, I was hoping to leave as is to minimize discrepancy (and means you can swap easily swap between the two)

Reply:
Yeah def appreciate the attention here, but for this case, I wouldn't treat this like a "new attribute" since it's already named/used elsewhere.